website/content/docs/commands/var/put.mdx

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+page_title: "Command: var put"
+description: |-
+  The "var put" command writes a variable to the given path in Nomad.
+---
+
+# Command: var put
+
+The `var put` command creates or updates an existing variable.
+
+## Usage
+
+```plaintext
+nomad var put [options] <variable spec file reference> [<key>=<value>]...
+nomad var put [options] <path to store variable> [<variable spec file reference>] [<key>=<value>]...
+```
+
+Variable metadata and items can be supplied using a variable specification, by
+using command arguments, or by a combination of the two techniques. An entire
+variable specification can be provided to the command via standard input (stdin)
+by setting the first argument to "-" or from a file by using an @-prefixed path
+to a variable specification file. When providing variable data via stdin, you
+must provide the `-in` flag with the format of the specification, either "hcl"
+or "json"
+
+Items to be stored in the variable can be supplied using the specification, as a
+series of key-value pairs, or both. The value for a key-value pair can be a
+string, an @-prefixed file reference, or a '-' to get the value from stdin. Item
+values provided from file references or stdin are consumed as-is with no
+additional processing and do not require the input format to be specified.
+
+Values supplied as command line arguments supersede values provided in the any
+variable specification piped into the command or loaded from file. If ACLs are
+enabled, this command requires the `variables:write` capability for the
+destination namespace.
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Put Options
+
+- `-check-index` `(int: <unset>)`: If set, the variable is only acted upon if
+  the server-side version's index matches the provided value. When a variable
```

Comment:
```suggestion
  the server-side version index matches the provided value. When a variable
```

Reply:
This is a possessive case: we're comparing the index that belongs to the version that belongs to the server with the index that belongs to the version to that belongs to the caller.